storage/client/file/src/test/resources/session-records/clearMetadataFromDirClient.json

```diff
@@ -1,42 +1,42 @@
 {
   "networkCallRecords" : [ {
     "Method" : "PUT",
-    "Uri" : "https://sima.file.core.windows.net/dirsharename/directory863334?restype=directory",
+    "Uri" : "https://azstoragesdkaccount.file.core.windows.net/dirsharename/directory149240?restype=directory",
```

Comment:
My comment came in rather late so sorry to keep harping on this if you've already fixed it and just haven't pushed yet :P 

I looked up the `azstoragesdkaccount` a few minutes ago, it's not listed in the test resources subscription. Please use a storage account in the test resources subscription. 

More info here: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki?pagePath=%2FData%20Plane%20Quality%20Program%2FTesting%20Guidelines&pageId=51&wikiVersion=GBwikiMaster&anchor=best-practices

Reply:
I am switching the account to Azure SDK Test Resources. This is the one created by our team and only used for java storage testing.